vraptor-site/layouts/header.html

```diff
@@ -9,7 +9,7 @@ <h1 class="logo"><a class="logo-link" href="/<%=path%>">
   <div class="header-links">
     <ul class="community">
 
-      <li><a href="https://groups.google.com/forum/<%=path == 'en' ? '?hl=en#!forum/vraptor4' : '#!forum/caelum-vraptor'%>" target="_blank">Google Groups</a></li>
+      <li><a href="https://groups.google.com/forum//#!forum/caelum-vraptor<%= path == 'en' ? '-en' : '' %>>" target="_blank">Google Groups</a></li>
```

Comment:
@danilomunoz thank you for changing this, but looks like it has an extra 
slash and greater-than sign. This is how the resulted URL is looking like:

> https://groups.google.com/forum//#!forum/caelum-vraptor-en>

and this is how it should look like:

> https://groups.google.com/forum/#!forum/caelum-vraptor-en